msi/GitAdditionalOptionsDlg.wxs

```diff
@@ -4,11 +4,17 @@
     <UI>
       <Dialog Id="GitAdditionalOptionsDlg" Width="370" Height="270" Title="!(loc.GitAdditionalOptionsDlg_Title)" X="50" Y="50">
         <Control Type="Text" Id="EnableFsCacheText" Width="275" Height="30" X="38" Y="72" TabSkip="no" Text="!(loc.GitAdditionalOptionsDlg_EnableFSCacheText)" />
-        <Control Type="CheckBox" Id="EnableFsCache" Width="324" Height="15" X="20" Y="53" CheckBoxValue="1" Integer="yes" Property="ENABLEFSCACHE" Text="!(loc.GitAdditionalOptionsDlg_EnableFsCacheCheckBox)" />
+        <Control Type="CheckBox" Id="EnableFsCache" Width="324" Height="15" X="20" Y="53" Integer="yes" CheckBoxValue="1" Property="UI_ENABLEFSCACHE" Text="!(loc.GitAdditionalOptionsDlg_EnableFsCacheCheckBox)" />
         <Control Type="Text" Id="EnableGcmText" Width="275" Height="40" X="38" Y="135" TabSkip="no" Text="!(loc.GitAdditionalOptionsDlg_EnableGcmText)" />
-        <Control Type="CheckBox" Id="EnableGCM" Width="324" Height="15" X="20" Y="116" CheckBoxValue="1" Integer="yes" Property="ENABLEGCM" Text="!(loc.GitAdditionalOptionsDlg_EnableGcmCheckBox)" />
+        <Control Type="CheckBox" Id="EnableGCM" Width="324" Height="15" X="20" Y="116" CheckBoxValue="1" Integer="yes" Property="ENABLEGCM" Text="Enable Git Credential Manager for Windows">
+          <Condition Action="disable">NOT WIX_IS_NETFRAMEWORK_451_OR_LATER_INSTALLED</Condition>
+          <!-- When converting this to a feature, use a CDATA with the condition 'Installed OR WIX_IS_NETFRAMEWORK_451_OR_LATER_INSTALLED' -->
+        </Control>
         <Control Type="PushButton" Id="Back" Width="56" Height="17" X="180" Y="243" Text="!(loc.WixUIBack)" />
-        <Control Type="PushButton" Id="Next" Width="56" Height="17" X="236" Y="243" Default="yes" Text="!(loc.WixUINext)" />
+        <Control Type="PushButton" Id="Next" Width="56" Height="17" X="236" Y="243" Default="yes" Text="!(loc.WixUINext)">
+          <Publish Property="ENABLEFSCACHE" Value="false"><![CDATA[UI_ENABLEFSCACHE <> 1]]></Publish>
+          <Publish Property="ENABLEFSCACHE" Value="true">UI_ENABLEFSCACHE = 1</Publish>
+        </Control>
         <Control Type="PushButton" Id="Cancel" Width="56" Height="17" X="304" Y="243" Text="!(loc.WixUICancel)" Cancel="yes">
```

Comment:
Only if I don't follow your advice to add the properties as Implement them. If I follow that advice and this suggestion, then the MSI won't build for many commits due to missing properties. And adding the UI "as I go" will also result in changes to `GitProduct.wxs` as the dialogs are added _and_ sequenced. Remember, WiX is like XML describing a database's row data and their relationships to one another.